pkg/web/templates/partials/account/account_list.html

```diff
@@ -1,3 +1,5 @@
+{{ $canEditAccounts := not .IsViewOnly }}
+{{- with .AccountsList -}}
```

Comment:
Here is how to extract the API data from the scene. WE'll have to write the type assertion function for every APIData type we add.

Reply:
I definitely wouldn't have known I'd need to include `-` inside the brackets on line 2.

Reply:
You don't have to technically - those are whitespace eliminators; so if you put `{{-` that will eat all the whitespace that comes before the `{{` and if you add `-}}` it will eat all the whitespace after the tag. If you didn't include them, the whitespace would remain there, which isn't really a big deal in HTML.

Reply:
Ah, ok.